hts-precompile/TokenManagementContract.sol

```diff
@@ -0,0 +1,491 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./FeeHelper.sol";
+
+contract TokenManagementContract is FeeHelper {
+
+    string name = "tokenName";
+    string symbol = "tokenSymbol";
+    string memo = "memo";
+    uint initialTotalSupply = 1000;
+    uint32 maxSupply = 1000;
+    uint decimals = 8;
+    bool freezeDefaultStatus = false;
+
+    event CreatedToken(address tokenAddress);
```

Comment:
Wrapper contracts should be simple pass-through contracts.  Having the contract emit these events while not emitting the same events via a direct call results in a bad user experience.

Reply:
Yes, you're right. I've removed those events.